### PR TITLE
Make a stars

### PR DESCRIPTION
**Pull request by Zhekich123**
DEMO LINK: https://zhekich123.github.io/layout_stars/
REPORT LINK: https://Zhekich123.github.io/layout_stars/report/html_report/

I faled 6 tests, but I have no idea why this blocks with stars have a difference with a reference.. 
I tried to move all blocks in the left side for 1-3px with margin, padding and just position: relative; right: 2px; and blocks is move successfully but difference is the same..

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+.stars-frame {
+  width: 96px;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```
Comment (IvanFesenko) [inline src/style.css:25]: Add right margin for stars except last one

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```
Comment (IvanFesenko) [inline src/style.css:67]: Do not repeat your self!
```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url(./images/star-active.svg);
}
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Check my comments

File: src/style.css (hunk 1 above)
Comment (Zhekich123) [reply src/style.css:25]: > Add right margin for stars except last one

I don't understand, I trying to add but it's still wrong..

**Pull request closed without merging**